scripts/common/fetch_primers.py

```diff
@@ -194,41 +268,43 @@ def organise_sars_cov_2_primers(source_schemes_path: Path, dest_schemes_path: Pa
 
 
 ORGANISE_PRIMERS = {
-    SARS_COV_2: organise_sars_cov_2_primers,
+    SARS_COV_2: {
+        EPI2ME_LABS: process_sars_cov_2_epi2me_labs_primers,
+        QUICK_LAB: process_sars_cov_2_quick_lab_primers,
+    },
 }
 
 
+def prepare_primers(
+    source_name: str,
+    primer_repo: str,
+    repo_commit: str,
+    pathogen: str,
+    source_schemes: str,
+    dest_schemes_path: Path,
+) -> SCHEME_TYPE:
+    # use a temporary directory for cloning the repository
+    with tempfile.TemporaryDirectory() as tmpdir:
+        print(f"Cloning repo {primer_repo} on {repo_commit}")
```

Comment:
Should the print stay or is this debugging logic left behind?

Reply:
Yeah, it stays. No harm, but at least the user knows what the script is doing. It's a standalone script.